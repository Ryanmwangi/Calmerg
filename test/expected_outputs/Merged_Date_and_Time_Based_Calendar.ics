BEGIN:VCALENDAR
PRODID:-//Your Product ID//EN
VERSION:2.0
NAME:Merged Date and Time Based Calendar
BEGIN:VEVENT
UID:20231225T000000-001@example.com
SUMMARY:Holiday_2023 Christmas Day
DTSTART:20231225T000000
DTEND:20231226T000000
END:VEVENT
BEGIN:VEVENT
UID:20231108T090000-001@example.com
SUMMARY:Work_Task Work Task
DTSTART:20231108T120000
DTEND:20231108T130000
END:VEVENT
END:VCALENDAR