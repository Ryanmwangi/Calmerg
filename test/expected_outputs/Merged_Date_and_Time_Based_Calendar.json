{
  "linkGroupName": "Merged Date and Time Based Calendar",
  "calendars": [
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\holiday_calendar_2023.ics",
      "prefix": "Holiday_2023",
      "override": false
    },
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\work_task_calendar.ics",
      "prefix": "Work_Task",
      "override": false
    }
  ]
}